vm-mock/vm.js

```diff
@@ -279,10 +279,66 @@ export default function createMockedABI(
           const addressStorage = ledger.get(contractAddress).storage;
           if (addressStorage.has(k)) {
             addressStorage.delete(k);
+          } else {
+            console.log('key not found');
           }
         }
       },
 
+      assembly_script_delete_data_for(addressPtr, keyPtr) {
+        const address = ptrToString(addressPtr);
+        const key = ptrToUint8ArrayString(keyPtr);
+
+        if (ledger.has(address)) {
+          const addressStorage = ledger.get(address).storage;
+          if (addressStorage.has(key)) {
+            addressStorage.delete(key);
+          } else {
+            console.log('key not found');
+          }
+        } else {
+          console.log('address not found');
+        }
+      },
+
+      assembly_script_append_data(keyPtr, valuePtr) {
+        const address = contractAddress;
+        const key = ptrToUint8ArrayString(keyPtr);
+        const newValue = byteArrToUTF8String(getArrayBuffer(valuePtr));
+
+        if (!ledger.has(address)) {
+          console.log('address not found');
```

Comment:
```suggestion
          console.log('address not found');
          return;
```

Reply:
I throw an error instead to match runtime